virtualization-host-formula/metadata/form.yml

```diff
@@ -25,7 +25,7 @@ default_pool:
     $name: Pool local path
     $default: /var/lib/libvirt/images
     $help: Path to the folder containing the disk images
-    $required: true
+    $required: default_pool_enabled == true
```

Comment:
As `default_pool_enabled` is of boolean type we could skip the comparison here but I see it in other places like this as well.